pkg/api/handlers/compat/networks.go

```diff
@@ -308,7 +308,7 @@ func CreateNetwork(w http.ResponseWriter, r *http.Request) {
 	}
 
 	opts := nettypes.NetworkCreateOptions{
-		IgnoreIfExists: !networkCreate.CheckDuplicate,
+		IgnoreIfExists: !networkCreate.CheckDuplicate, //nolint:staticcheck
```

Comment:
why nolint?

Reply:
The comment stated that it's now always `true` so it'll be false going forward.  It felt a bit too strange so I just let it as is.

Reply:
Reading the comment the CheckDuplicate should now be on by default as of API version 1.44 but our code does not handle it at all. So the proper thing would be to add this logic to match docker.

Reply:
Can you or somebody else take over?  I won't find time to push it over the finish line.

Reply:
I think it does the right thing now.  Also added a comment.

Reply:
you need to fix the API tests that checked this behaviour.
I still think it safer to to the version check and still respect this field for 1.43 and older but then again I doubt anyone actually depends on this behavior so I guess I am fine with the change as is once you remove the old test for it.

Reply:
Done.
@mheon PTAL, this is a breaking change for 5.0 (following Docker compat)